### PR TITLE
Porting Rosetta problems

### PR DESCRIPTION
**Pull request by NitronR**

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
@NitronR, took at look at all of your ports. I just found a couple of things. The first is that the problem "General FizzBuzz" is missing `/// description:` before the actual problem description, causing an error. Also, parts of "Identity matrix" and "Jaro distance" are not rendered properly. Other than that, everything LGTM!

**Comment by NitronR** [comment]
@scissorsneedfoodtoo Fixed the mentioned ports. Thanks for checking.

**Comment by scissorsneedfoodtoo** [comment]
@NitronR, no problem! Any time.

File: client/scripts/challenges/rosettacode/formatted/G/Gaussian elimination.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+/// categories:
+/// Matrices
+
+/// difficulty: 5
+
+/// benchmark:
+
+/// description:
+/// <div class="rosetta">
+/// <p class="rosetta__paragraph">Write a function to solve Ax=b using Gaussian elimination then backwards substitution. A being an n by n matrix. Also, x and b are n by 1 vectors. To improve accuracy, please use partial pivoting and scaling.</p></div>
```
Comment (alvinkl) [inline client/scripts/challenges/rosettacode/formatted/G/Gaussian elimination.js:20]: Hi @NitronR, can you wrap code element for the Math equation? so it looks nicer.

Like ```Ax=b```
screen shot 2018-07-02 at 08 58 49

or bold also works, just like the original one.
not a big issue though, just nice to have

**Comment by alvinkl** [comment]
@NitronR Can you fix the one I commented? Everything looks good,
Btw can you merge squash all of the commits? so if anything happens we just need to revert 1 commit.
Other than that code should be LGTM

**Comment by NitronR** [comment]
@alvinkl The problem seeds are moved to the freeCodeCamp/curriculum [here](https://github.com/freeCodeCamp/curriculum/blob/dev/challenges/08-coding-interview-prep/rosetta-code.json). The equation in the description is styled correctly there. Should I close this PR as these problems are already included there along with many other new problems?
Thanks for the review though!!

**Comment by NitronR** [comment]
Moved to curriculum

**Pull request closed without merging**